### PR TITLE
how-to/monitor: update manual deploy monitor

### PR DESCRIPTION
**Pull request by liubo0127**
### What is changed, added or deleted? 

Update the dashboard file for manual deploy monitor. In master or 3.0, some file does not exist.

### What is the related PR or file link(s)? 

https://github.com/pingcap/tidb-ansible/issues/999
#1962

### Which version does your change affect? 

Affects TiDB master and 3.0.

### CODE REVIEW

File: dev/how-to/monitor/monitor-a-cluster.md [hunk 1]
```diff
@@ -314,7 +314,7 @@ url = https://grafana.net
 
 3. 点击 **Upload .json File** 上传对应的 JSON 文件（下载 [TiDB Grafana 配置文件](https://github.com/pingcap/tidb-ansible/tree/master/scripts))。
 
-    > **注意**：TiKV、PD 和 TiDB 面板对应的 JSON 文件分别为 `tikv_pull.json`，`pd.json`，`tidb.json`。
+    > **注意**：TiKV、PD 和 TiDB 面板对应的 JSON 文件分别为 `tidb_summary.json`，`tikv_details.json`，`tikv_summary.json`，`pd.json`，`tidb.json`，`tidb_summary.json`。
```
Comment (lilin90) [inline dev/how-to/monitor/monitor-a-cluster.md]: - Two duplicate `tidb_summary.json` files
- Inappropriate order

```suggestion
    > **注意**：TiKV、PD 和 TiDB 面板对应的 JSON 文件分别为 `tikv_summary.json`，`tikv_details.json`，`tikv_trouble_shooting.json`，`pd.json`，`tidb.json`，`tidb_summary.json`。
```

File: v3.0/how-to/monitor/monitor-a-cluster.md [hunk 2]
```diff
@@ -315,7 +315,7 @@ url = https://grafana.net
 
 3. 点击 **Upload .json File** 上传对应的 JSON 文件（下载 [TiDB Grafana 配置文件](https://github.com/pingcap/tidb-ansible/tree/master/scripts))。
 
-    > **注意**：TiKV、PD 和 TiDB 面板对应的 JSON 文件分别为 `tikv_pull.json`，`pd.json`，`tidb.json`。
+    > **注意**：TiKV、PD 和 TiDB 面板对应的 JSON 文件分别为 `tidb_summary.json`，`tikv_details.json`，`tikv_summary.json`，`pd.json`，`tidb.json`，`tidb_summary.json`。
```
Comment (lilin90) [inline v3.0/how-to/monitor/monitor-a-cluster.md]:
```suggestion
    > **注意**：TiKV、PD 和 TiDB 面板对应的 JSON 文件分别为 `tikv_summary.json`，`tikv_details.json`，`tikv_trouble_shooting.json`，`pd.json`，`tidb.json`，`tidb_summary.json`。
```

**Comment by liubo0127** [comment]
> * The changes in this PR is different from that comment in [pingcap/tidb-ansible#999](https://github.com/pingcap/tidb-ansible/issues/999). Please check whether we need to add all JSON files mentioned in that comment. For example,  is missing.
> * The `best-practices` label only applies to best practice articles. This is not one, so you don't need to add that label.

You are right, `tikv_trouble_shooting.json` is indeed missing.

**Comment by lilin90** [comment]
@liubo0127 I think you might need a technical review?

File: dev/how-to/monitor/monitor-a-cluster.md [hunk 3]
```diff
@@ -314,7 +314,7 @@ url = https://grafana.net
 
 3. 点击 **Upload .json File** 上传对应的 JSON 文件（下载 [TiDB Grafana 配置文件](https://github.com/pingcap/tidb-ansible/tree/master/scripts))。
 
-    > **注意**：TiKV、PD 和 TiDB 面板对应的 JSON 文件分别为 `tikv_pull.json`，`pd.json`，`tidb.json`。
+    > **注意**：TiKV、PD 和 TiDB 面板对应的 JSON 文件分别为 `tikv_summary.json`，`tikv_details.json`，`tikv_trouble_shooting.json`，`pd.json`，`tidb.json`，`tidb_summary.json`。
```
Comment (lilin90) [inline dev/how-to/monitor/monitor-a-cluster.md]:
```suggestion
    > **注意：**
    >
    > TiKV、PD 和 TiDB 面板对应的 JSON 文件分别为 `tikv_summary.json`，`tikv_details.json`，`tikv_trouble_shooting.json`，`pd.json`，`tidb.json`，`tidb_summary.json`。
```

File: v3.0/how-to/monitor/monitor-a-cluster.md [hunk 4]
```diff
@@ -315,7 +315,7 @@ url = https://grafana.net
 
 3. 点击 **Upload .json File** 上传对应的 JSON 文件（下载 [TiDB Grafana 配置文件](https://github.com/pingcap/tidb-ansible/tree/master/scripts))。
 
-    > **注意**：TiKV、PD 和 TiDB 面板对应的 JSON 文件分别为 `tikv_pull.json`，`pd.json`，`tidb.json`。
+    > **注意**：TiKV、PD 和 TiDB 面板对应的 JSON 文件分别为 `tikv_summary.json`，`tikv_details.json`，`tikv_trouble_shooting.json`，`pd.json`，`tidb.json`，`tidb_summary.json`。
```
Comment (lilin90) [inline v3.0/how-to/monitor/monitor-a-cluster.md]:
```suggestion
    > **注意：**
    >
    > TiKV、PD 和 TiDB 面板对应的 JSON 文件分别为 `tikv_summary.json`，`tikv_details.json`，`tikv_trouble_shooting.json`，`pd.json`，`tidb.json`，`tidb_summary.json`。
```

**Pull request merged**